taskfiles/lint-yaml.yaml

```
@@ -13,16 +13,24 @@ tasks:
       - "{{.TASKFILE}}"
       - exclude: "{{.ROOT_DIR}}/**/build/*"
       - exclude: "{{.ROOT_DIR}}/**/tools/*"
+      # The following are for Clion's default generated build dirs
+      - exclude: "{{.ROOT_DIR}}/cmake-build-debug/*"
+      - exclude: "{{.ROOT_DIR}}/cmake-build-release/*"
```

```suggestion

      # Exclude CMake build directories generated by CLion
      - exclude: "{{.ROOT_DIR}}/cmake-build-*"
```

* We should include an empty line before each comment block.
* CLion can generate many different build directories, but they're all prefixed by `cmake-build-`